src/main/java/org/owasp/benchmark/testcode/BenchmarkTest00524.java

```diff
@@ -85,17 +85,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 //		byte[] iv = random.generateSeed(16);
 		
 		try {
-		    java.util.Properties benchmarkprops = new java.util.Properties();
-		    benchmarkprops.load(this.getClass().getClassLoader().getResourceAsStream("benchmark.properties"));
+			java.util.Properties benchmarkprops = new java.util.Properties();
+			benchmarkprops.load(this.getClass().getClassLoader().getResourceAsStream("benchmark.properties"));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the resource returned by the following method call: *getResourceAsStream*. Currently, there are execution paths that do not contain closure statements, e.g., when exception is thrown by Writer.write. Either a) close the object returned by *getResourceAsStream()* in a try-finally block or b) close the resource by declaring the object returned by *getResourceAsStream()* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).